pylearn2/training_algorithms/learning_rule.py

```diff
@@ -258,3 +259,83 @@ def get_updates(self, learning_rate, grads, lr_scalers=None):
 
         return updates
 
+
```

Comment:
Everything in here needs docstrings.

Does anybody have any idea how why devtools/tests/test_format.py tested? This file isn't on the whitelist as far as I can tell.